cmd/httpd/handlers/auth.go

```diff
@@ -102,66 +101,46 @@ func (a *Auth) Login(w http.ResponseWriter, r *http.Request) {
 }
 
 func (a *Auth) Refresh(w http.ResponseWriter, r *http.Request) {
-	var request validateTokenRequest
-
-	decoder := json.NewDecoder(r.Body)
-	err := decoder.Decode(&request)
-	if err != nil {
-		a.logger.Error().Err(err).Msg("Unable to decode login data")
-
-		w.WriteHeader(http.StatusInternalServerError)
-		return
-	}
+	claims := r.Context().Value("claims").(jwt.MapClaims)
 
-	// TODO token deprecation probably should be done in transaction
-	oldToken, err := a.tokens.GetNotExpiredTokenByToken(request.Token)
+	uuid, err := uuid.Parse(claims["id"].(string))
```

Comment:
It would be nice before issuing new token to check that record with id and email exists in db

Reply:
fixed